src/main/scala/firrtl/stage/phases/ConvertCompilerAnnotations.scala

```diff
@@ -0,0 +1,40 @@
+// See LICENSE for license details.
+
+package firrtl.stage.phases
+
+import firrtl.AnnotationSeq
+import firrtl.options.{Dependency, OptionsException, Phase}
+import firrtl.stage.{CompilerAnnotation, RunFirrtlTransformAnnotation}
+
+@deprecated(
+  "This only exists to convert deprecated CompilerAnnotations to RunFirrtlTransformAnnotations.",
+  "FIRRTL 1.5.0"
+)
+class ConvertCompilerAnnotations extends Phase {
```

Comment:
Obviously this will add deprecation warnings to normal compilation, is it possible to make it package private and have this still work? Is that a reasonable thing to do?

Reply:
I can change it. I had it like this originally and changed it back because my `firrtlTests` test couldn't see it (yes, I should've just changed the test package...).

Reply:
Yeah I'd prefer it being private and not deprecated if possible haha, but I feel you

Reply:
Fixed in later commits.